components/draw/draw.service.ts

```diff
@@ -263,7 +263,9 @@ export class HsDrawService {
    */
   addDrawLayer(layer: Layer): void {
     this.HsMapService.map.addLayer(layer);
-    this.fillDrawableLayers();
+    setTimeout(() => {
```

Comment:
Why do we need timeout? Can you add a note please, so we dont remove it later?

Reply:
Oh, most likely we dont. There was a problem with selectedLayer interaction inside fillDrawable layers. At first I thought it just need to wait a bit before refilling..turned out to be wrong condition. I just pushed to test on server and then forgot to delete and squash. Most likely because I thought it doesnt really matter in this specific case

Reply:
Had to redo the condition once more, wasnt quite working still.  https://github.com/hslayers/hslayers-ng/pull/1373/commits/d592b4fe28c7c289b7a73b7ed2a5322b6faeb79c